.bazelrc

```diff
@@ -44,6 +45,15 @@ build:remote-dev --config=remote-shared
 build:remote-dev --config=cache-dev
 build:remote-dev --remote_executor=grpcs://remote.buildbuddy.dev
 
+# Build with --config=remote-mac-x to run cross platform builds
+# and tests remotely on Linux executors from a MacOS machine.
+build:mac-x --cpu=k8
+build:mac-x --host_cpu=k8
+build:remote-mac-x --config=mac-x
+build:remote-mac-x --config=remote
+build:remote-mac-x-dev --config=mac-x
+build:remote-mac-x-dev --config=remote-dev
```

Comment:
nit: maybe call this `mac-crossplatform` or even just `linux-target` (since this is really just specifying linux as the target platform)? If folks want a shorter alias like `mac-x` then they can use `user.bazelrc` for that.